samples/ClassroomLabs/Modules/Library/Az.LabServices.psm1

```diff
@@ -1,11 +1,14 @@
 # TODO: consider polling on the operation returned by the API in the header as less expensive for RP
 # TODO: consider creating proper PS1 documentation for each function
 
-# We are using strict mode for added safety
-Set-StrictMode -Version Latest
+# Only set these if not running under Azure Automation
+if ("AzureAutomation/" -ne $env:AZUREPS_HOST_ENVIRONMENT) {
+    # We are using strict mode for added safety
+    Set-StrictMode -Version Latest
 
-# We require a relatively new version of Powershell
-#requires -Version 3.0
+    # We require a relatively new version of Powershell
+    #requires -Version 3.0
+}

```

Comment:
The commandlet and directive seem to work differently in Azure Automation than in regular PowerShell.  The only way I could make it work is excluding these lines if we're running in an Azure Automation environment...

Reply:
Weird.  Honestly, do we even need this anymore?

Reply:
I don't think so - I haven't heard of any customers on super old versions of PowerShell any longer...  I'm fine removing it!